examples/00-Different-analysis-types/05-fluid-simulation.py

```diff
@@ -29,66 +29,75 @@
-# ----------------
-###############################################################################
-# Check the metadata on available results
+# Check metadata on available results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 print(simulation.result_info)
 
 ###############################################################################
-# Print a specific one to get more information on available qualifiers (zones, phases and so on)
+# Extract a result
+# ----------------
+# Print a specific result to get more information on available qualifiers
+# (such as zones and phases).
 print(simulation.result_info["enthalpy"])
 # Or use an index
 # print(simulation.result_info[0])
 
 ###############################################################################
-# Extract this available result as a Dataframe
+# Extract this result as a dataframe
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 enthalpy = simulation.enthalpy()
 print(enthalpy)
 # Not specifying any qualifier returns a unique column of data
 
 ###############################################################################
-# Plot the Dataframe
+# Plot dataframe
+# ~~~~~~~~~~~~~~
 enthalpy.plot()
 
 ###############################################################################
-# Available qualifiers for this result can be used in the extraction request
-# to filter/separate data.
-# Here we want a different column for each available zone for this result
+# Use qualifiers for result to filter or separate data
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# You can use available qualifiers for the result in the extraction request
+# to filter or separate data.
+#
+# This code uses a different column for each available zone for the result.
 print(simulation.enthalpy(zone_ids=[13, 28]))
-# Here only the data for the fluid-stator zone is extracted
+#
+# This code extracts only the data for the fluid-stator zone.
 print(simulation.enthalpy(zone_ids=[28]))
-# The same logic can be applied to any available qualifier found in the ``AvailableResult``
+#
+# You can apply the same logic to any available qualifier found in the ``AvailableResult``
 # description under 'Available qualifier labels'.
 
 ###############################################################################
-# The result extraction request can also contain selection arguments
-# The enthalpy result being defined on cells, you can request data for specific
-# cells using their IDs:
+# The result extraction request can also contain selection arguments.
+# Because the enthalpy result is being defined on cells, you can request data
+# for specific cells using their IDs:
 print(simulation.enthalpy(cell_ids=[1, 2]))
 
 ###############################################################################
-# For selection and manipulation of the Dataframe,
-# please refer to example "Create and manipulate a DPF Dataframe"
+# For information on selection and manipulation of the dataframe,
+# see the "Create and manipulate a DPF dataframe" example.
```

Comment:
Where is the referenced example? We should make it a link.

Reply:
@PipKat you can find it [here](https://github.com/ansys/pydpf-post/blob/master/examples/01-Detailed-Examples/00-dataframe-manipulation.py)

Reply:
@PProfizi Make the reference to the example a link.